### DISCUSSION TITLE
Santoker R3 Bluetooth/Wifi Connectivity Issues

### DISCUSSION THREAD

**Discussion by bicibarista** [artisan-roaster-scope/artisan#1491] [Q&A]
Hello, I am trying to set up Artisan to work with my new Santoker R3 roaster, when I go config>machines>santoker>R network, I am not getting any readings from ET/BT, what could be the issue?

**Comment by poundy** [comment]
https://artisan-scope.org/machines/santoker/ and the code tell me your R model should have IP address of 10.10.100.254. Is that what your machine reports? Is your PC's IP the same network zone as that?

**Comment by MAKOMO** [comment]
Your machine might communicate via Bluetooth and not WiFi. As all Santoker R models produced after 10/2023.

[Santoker R Series support](https://artisan-scope.org/machines/santoker/) was just improved and Bluetooth communication was added. Could you please check if the setup `R Series Bluetooth` (menu `Config >> Machines >> Santoker`) of the current [Continuous Build](https://github.com/artisan-roaster-scope/artisan/releases/tag/continuous) works with your machine (assuming it is not a R Master Series and it features Bluetooth communication).
